### PR TITLE
[Wisp] Jstack info for containerId added

### PR DESCRIPTION
**Pull request by Sere-Fu**
Test Plan: jreg pipeline
Reviewed-by: yulei, joeyleeeeeee97, zhengxiaolinX
Issue: alibaba/dragonwell8#235

### CODE REVIEW

File: src/share/vm/classfile/javaClasses.cpp [hunk 1]
```diff
@@ -977,6 +978,11 @@ oop java_lang_Thread::name(oop java_thread) {
 }
 
 
+oop java_lang_Thread::resourceContainer(oop java_thread) {
+  return java_thread->obj_field(_resourceContainer_offset);
+}
+
+
```
Comment (joeyleeeeeee97) [inline src/share/vm/classfile/javaClasses.cpp:985]: please remove extra blank line

File: src/share/vm/classfile/javaClasses.hpp [hunk 2]
```diff
@@ -1563,6 +1565,15 @@ class com_alibaba_wisp_engine_WispControlGroup_CpuLimit: AllStatic {
   static void compute_offsets();
 };
 
+class com_alibaba_rcm_internal_AbstractResourceContainer: AllStatic {
+private:
+  static int _id_offset;
+public:
+  // static void serialize_offsets(SerializeClosure* f) NOT_CDS_RETURN;
```
Comment (joeyleeeeeee97) [inline src/share/vm/classfile/javaClasses.hpp:1572]: remove this?

File: src/share/vm/runtime/coroutine.cpp [hunk 3]
```diff
@@ -502,13 +502,19 @@ oop Coroutine::print_stack_header_on(outputStream* st) {
...
     char buf[128] = "<cached>";
     if (thread_obj != NULL) {
       oop name = java_lang_Thread::name(thread_obj);
       if (name != NULL) {
         java_lang_String::as_utf8_string(name, buf, sizeof(buf));
       }
+      oop container = java_lang_Thread::resourceContainer(thread_obj);
+      if (container != NULL) {
+        cgId = com_alibaba_rcm_internal_AbstractResourceContainer::get_id(container);
+        // cgId = 3;
```
Comment (joeyleeeeeee97) [inline src/share/vm/runtime/coroutine.cpp:516]: remove commented code

**Comment by joeyleeeeeee97** [comment]
Also, some changes in escape analysis seem unnecessary

File: src/share/vm/classfile/javaClasses.cpp (hunk 1 above)
Comment (Sere-Fu) [reply src/share/vm/classfile/javaClasses.cpp:985]: seems no need to remove(2 blank lines between functions)

**Comment by joeyleeeeeee97** [comment]
Please add a test for this?

**Comment by Sere-Fu** [comment]
> Please add a test for this?

test added.

**Review by joeyleeeeeee97 — APPROVED** [review]
LGTM

**Comment by yuleil** [comment]
image
The commit log format is incorrect. Fix it please.

**Review by yuleil — APPROVED** [review]

**Review by zhengxiaolinX — APPROVED** [review]

**Pull request merged**